domains/ramiro.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "email": "urteagaramiro33@gmail.com"
+    },
+    "records": {
+      "CNAME": "https://portafolio-chi-gilt.vercel.app"
```

Comment:
CNAME records must be hostnames with no protocols. See guide on how to use vercel for domains: https://docs.is-a.dev/guides/vercel